mackerel-plugin-solr/lib/solr.go

```diff
@@ -122,7 +133,15 @@ func (s *SolrPlugin) loadStats() error {
 		if err != nil {
 			return err
 		}
-		err = s.loadStatsMbeanQueryHandler(core)
+		err = s.loadStatsMbeanHandler(core, "QUERYHANDLER")
+		if err != nil {
+			return err
+		}
+		err = s.loadStatsMbeanHandler(core, "UPDATEHANDLER")
```

Comment:
I understand as following, is it right?:
- On Solr 6.x, some stats in `QUERYHANDLER` moved to `UPDATEHANDLER` and `REPLICATION`
- On Solr 5.x, we cannot get desired stats from `UPDATEHANDLER` and `REPLICATION`, so simply ignoring their results.